### PR TITLE
Created srivatsavsurya.mdx

### PR DESCRIPTION
**Pull request by srivatsavsurya**

### CODE REVIEW

**Review by sboy99 — CHANGES_REQUESTED** [review]
Add some content too, Your can view other contributors profile for reference

**Comment by priyankarpal** [comment]
@srivatsavsurya  add your details such as skills, name, socials links etc.

**Comment by srivatsavsurya** [comment]
Check now

File: contribution/srivatsavsurya.mdx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+github_username: srivatsavsurya
+occupation: Student
+github: https://github.com/srivatsavsurya
+twitter: https://twitter.com/srivatsavsurya
+linkedin: https://www.linkedin.com/in/srivatsav
+---
+
+# Introduction
+
+Hi👋 there! It's srivatsav, a Third-year MEchatronics Science Engineering student & a Tech + Mechanics Enthusiast. I'm a Student Engineer.I am always looking for ways to contribute to the open-source community and positively impact the tech industry.
```
Comment (priyankarpal) [inline contribution/srivatsavsurya.mdx:12]:
``` suggestion
Hi👋 there! It's Srivastava, a Third-year Mechatronics Science Engineering student & a Tech + Mechanics Enthusiast. I'm a Student Engineer. I am always looking for ways to contribute to the open-source community and positively impact the tech industry.
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]
Just small changes

**Review by sboy99 — APPROVED** [review]
LGTM

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**